scripts/common.py

```diff
@@ -79,6 +80,7 @@
 RELEASE_MAINTENANCE_BRANCH_REGEX = r'release-maintenance-(\d+\.\d+\.\d+)$'
 HOTFIX_BRANCH_REGEX = r'release-(\d+\.\d+\.\d+)-hotfix-[1-9]+$'
 TEST_BRANCH_REGEX = r'test-[A-Za-z0-9-]*$'
+MAX_WAIT_TIME_FOR_PORT_TO_OPEN_SECS = 60
```

Comment:
on run_e2e test script it's 1000, should we add a named param `wait_time` with dafault value MAX_WAIT_TIME_FOR_PORT_TO_OPEN_SECS for `wait_for_port_to_be_open`?